src/components/navbar/Navbar.tsx

```diff
@@ -41,13 +42,17 @@ const logoContainer = classnames(
   margin('mt-2')
 )
 
-const logoWithVersion = classnames(display('flex'), flexDirection('flex-col'))
+const logoWithVersion = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  displayFrom('md')
+)
 
-const logoWrapper = classnames(display('flex'), width('w-1/4', 'sm:w-full'))
+const logoWrapper = classnames(display('flex'), width('w-full'))
 
 export default function () {
   const { pathname } = useLocation()
-  const withoutWallet = pathname.split('/').length >= 3
+  const withoutRightBlock = pathname.split('/').length >= 3
```

Comment:
Maybe it's better to make it via `Layout`? But I guess it should be a separate task...

Reply:
Let's make a task for it